aplikigo-1/tasks/phase-3/task-1/quest-opposed-guerilla.json

```diff
@@ -0,0 +1,19 @@
+{
+    "valOprAddr": "regen:valoper1lts57fa2sr0vn6swyl30rmwsmjwzupaaqe767t",
+    "codeId": ["119", "130", "131"],
+    "contractAddress": ["regen:1xdu908l2thfzutvazzht09m6y3qmnx5kpzdtw8", "regen:1w462vak4furdk0mnjfkyx746u2n7exn9tfmnuz", "regen:14tzrf0w2j2u4muyqtdr4l7jgs9m0q04syuyx46"] ,
+    "mintTxHash": "471939664D93C938EEA2EBCE083CE672AA174D6D3DC56331CDB4BD05B4B9866B",
+    "transferTxHash": ["FA32957363CB55C6FC1E260832281AAD96F10DDEE0649033924EA92063B13964", "C13EAD5C2FC0A74DB18F2C5D43C06B5BF592BFA8F68007BC1729567FE6A91A49"],
+    "otherTxs": {
+        "burn": ["3304A5295430DC9BBC72654CC384EE139E2886EA0F5B569CC2B9409AD6E3C236", "hash2"],
+        "send": ["hash1", "hash2"],
+        "burnFrom": ["AA01B35721CAECF4129C07B44B7B31589BE14DA30555C605A5E533F03A0978CF", "7FB0F534513AFC6C3005233BDC2DD80D1A9FE1D0A2A8E0EAE1B10F1DEF20CAE0"],
+        "transferFrom": ["F4ACB149F303F11C1EDAAA2C6E2A232A22C679F05B69D3D726BB2CB8B0173941", "172456C1E276B8447D55BAB7522F1CE3195068E34C41EE5973ACEB7BA963AACF"],
+        "increaseAllowance": ["77E74A93027B711E06E896BBA038759EAEF39BBB0F46FD1895D0F014DF2E9877", "15D171FE34D63C3E3536B954F1803A9C70E0FEEE729AEC79CA0DACA2EF98970A"],
+        "decreaseAllowance": ["8DDCEBAC5CE5DF1ED5DFB3C7653413332400E317D6A9C30B4B357C058AFDEF84", "F919511A4197CF5D037B66097775DCD18B2F28BBC2D7450C6002BCE93D49E301"],
+    },
+    "customTxs":{
+		"something": "A2680C9A5A2DB131B13248AB2F9F47C94BD6EF5F2A6D94A95ADE3101AA2EB919",
+		"nothing": "993844E5D857D7DBDD5C5CF3B86753CE6EB5FC495977A9A6DA25807F117E230E"
+    }
+}
```

Comment:
custom methods `something` and `nothing` return an attribute `note` with values `nice` and `not nice`, respectively. nothing fancy.